src/component/admin/groups/GroupsList/GroupCard/GroupCardAvatars/GroupCardAvatars.tsx

```diff
@@ -42,6 +43,20 @@ export const GroupCardAvatars = ({ users }: IGroupCardAvatarsProps) => {
         () => users.sort((a, b) => (a.role < b.role ? 1 : -1)).slice(0, 9),
         [users]
     );
+
+    const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+    const [popupUser, setPopupUser] = useState<IGroupUser>();
+
+    const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
```

Comment:
We've standardized on using the naming convention of `onPopoverOpen` for event handlers

Reply:
Ah good catch. Fixed now.